codegen/src/test/resources/java/org/com/generated/contracts/Greeter.java

```diff
@@ -0,0 +1,178 @@
+package org.com.generated.contracts;
```

Comment:
Surely generated code shouldn't be in here?

Reply:
this will be changed to org.com.test.contract this is a contract I need for the tests